packages/terra-functional-testing/tests/jest/commands/utils/eventEmitter.test.js

```diff
@@ -1,5 +1,5 @@
-import eventEmitter from '../../../../src/commands/utils/eventEmitter'; // eslint-disable-line import/no-duplicates
-import eventEmitter2 from '../../../../src/commands/utils/eventEmitter'; // eslint-disable-line import/no-duplicates
+const eventEmitter = require('../../../../src/commands/utils/eventEmitter'); // eslint-disable-line import/no-duplicates
```

Comment:
Just curious, why do we need to change to `require` syntax here?

Reply:
For some reason, babel was having issues with compiling `import` at the package level while mysteriously working at the root level. I had theoretically the same config on both levels so I couldn't figure out what's going on. However during debugging, I discover that most of the terra-functional-tests tests utilize `require` and only the ones with `import` were causing issues. So I changed these for consistency and it resolved the babel issue.